src/patterns/components/inputs/date-disabled.hbs

```diff
@@ -6,7 +6,7 @@ information:
   - Spark recommends that this component be used when you expect the entered date to be in the past.
   - Masking is automatically applied when the field contains a valid input.
   - The field value contains special characters which you may need to parse out before submitting the form.
-order: 28
+order: 30
```

Comment:
I would double check that with my reordering that this is the right order. I don't remember 100% but I thought they only incremented by 1 and I see 2 here. I'm not sure.